src/index.ts

```diff
@@ -139,3 +139,99 @@ export class DynamoDBAutoIncrement extends BaseDynamoDBAutoIncrement {
     return { puts, nextCounter }
   }
 }
+
+/**
+ * Update a history table with an auto-incrementing attribute value in DynamoDB
+ *
+ * @example
+ * ```
+ * import { DynamoDB } from '@aws-sdk/client-dynamodb'
+ * import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb'
+ * import { DynamoDBHistoryAutoIncrement } from '@nasa-gcn/dynamodb-autoincrement'
+ *
+ * const client = new DynamoDB({})
+ * const doc = DynamoDBDocument.from(client)
+ *
+ * const autoIncrementHistory = DynamoDBHistoryAutoIncrement({
+ *   doc,
+ *   counterTableName: 'widgetsHistory',
+ *   counterTableKey: {
+ *     widgetID: 42
+ *   },
+ *   attributeName: 'version',
+ *   tableName: 'widgets',
```

Comment:
For closer similarity to the behavior of the `DynamoDBAutoIncrement` class, the table named `counterTableName` should be the table that stores the current version number, and hence also the current item. The table named `tableName` should be the one that accumulates prior versions. Please switch the names of these two tables in your example.

With that change, do you need to update the implementation as well?

Reply:
Hm likely yes, I'll make some changes locally and see what all gets shifted around